src/stylesheets/components/_header.scss

```diff
@@ -193,7 +193,7 @@ $z-index-nav:     9000;
       &[aria-expanded=false],
       &[aria-expanded=true] { /* stylelint-disable-line selector-no-qualifying-type */
         @include media($nav-width) { /* stylelint-disable-line selector-no-qualifying-type */
-          background-position: right 1.5rem top 50%;
+          background-position: right 1.5rem top 44%;
```

Comment:
I thought based on the conversation, @maya was saying the nav wasn't meant to be vertically centered liked that, so I actually purposely didn't do it.